conformance_tests/core/test_memory/src/test_virtual_memory.cpp

```diff
@@ -156,33 +169,58 @@ TEST_F(
 
   for (auto accessFlags : memoryAccessFlags) {
     lzt::virtual_memory_map(context, reservedVirtualMemory, allocationSize,
-                            reservedPhysicalMemory, 0, accessFlags);
+                            reservedPhysicalDeviceMemory, 0, accessFlags);
+    lzt::virtual_memory_unmap(context, reservedVirtualMemory, allocationSize);
+  }
+  for (auto accessFlags : memoryAccessFlags) {
+    lzt::virtual_memory_map(context, reservedVirtualMemory, allocationSize,
+                            reservedPhysicalHostMemory, 0, accessFlags);
     lzt::virtual_memory_unmap(context, reservedVirtualMemory, allocationSize);
   }
 
-  lzt::physical_memory_destroy(context, reservedPhysicalMemory);
+  lzt::physical_memory_destroy(context, reservedPhysicalHostMemory);
+  lzt::physical_memory_destroy(context, reservedPhysicalDeviceMemory);
   lzt::virtual_memory_free(context, reservedVirtualMemory, allocationSize);
 }
 
 void RunGivenMappedReadWriteMemoryThenFillAndCopyWithMappedVirtualMemory(
-    zeVirtualMemoryTests &test, bool is_immediate) {
+    zeVirtualMemoryTests &test, bool is_host_memory, bool is_immediate) {
   auto bundle = lzt::create_command_bundle(test.device, is_immediate);
 
-  lzt::query_page_size(test.context, test.device, test.allocationSize,
-                       &test.pageSize);
+  if (is_host_memory) {
+    test.pageSize = 1UL << 21;
```

Comment:
do we need to set test.pageSize here?

Reply:
We need to because at line 197 we align the allocation size to page size. But this is a good point, you've reminded me that currently the test is Linux-only. I guess I need to restructure this so the host memory test is Linux-only, and use `sysconf(_SC_PAGESIZE)` to get page size.